src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaJsonWriter.Async.cs

```diff
@@ -0,0 +1,1722 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmModelCsdlSchemaJsonWriter.Async.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.OData.Edm.Csdl.CsdlSemantics;
+using Microsoft.OData.Edm.Csdl.Parsing.Ast;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OData.Edm.Vocabularies.V1;
+
+namespace Microsoft.OData.Edm.Csdl.Serialization
+{
+    /// <summary>
+    /// OData Common Schema Definition Language (CSDL) JSON writer
+    /// </summary>
+    internal partial class EdmModelCsdlSchemaJsonWriter : EdmModelCsdlSchemaWriter

```

Comment:
Accross this file: Other than for abbreviations (e.g. CSDL and JSON) please use sentence case in the docs for class, methods, and parameters